### PR TITLE
Wipe Workspace On Start

### PR DESCRIPTION
**Pull request by Bentlybro**
### Background
There was a request for "WIPE_WORKSPACE_ON_START" #4804 and i went ahead and added that

### Changes
added wipe_workspace_on_start: bool, set the default to be False and added the function to remove all the files inside the workspace folder while keeping the original workspace folder

### Test Plan
After adding this ran autogpt with "wipe_workspace_on_start" set to false and it skipped over this, when "wipe_workspace_on_start" is set to true it removes all the files inside the workspace folder and keeps the workspace folder

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have not snuck in any "extra" small tweaks changes.
- [x] I have run the following commands against my code to ensure it passes our linters:
    ```shell
    black .
    isort .
    mypy
    autoflake --remove-all-unused-imports --recursive --ignore-init-module-imports --ignore-pass-after-docstring autogpt tests --in-place
    ```
all tests passed

### CODE REVIEW

File: autogpt/config/config.py [hunk 1]
```diff
@@ -205,6 +207,19 @@ def build_config_from_env(cls):
                 "disabled_command_categories"
             ] = disabled_command_categories.split(",")
 
+        wipe_workspace_on_start = os.getenv("WIPE_WORKSPACE_ON_START")
```
Comment (lc0rp) [inline autogpt/config/config.py]:
```suggestion
        wipe_workspace_on_start = os.getenv("WIPE_WORKSPACE_ON_START", "False") == "True"
```

File: autogpt/config/config.py [hunk 2]
```diff
@@ -205,6 +207,19 @@ def build_config_from_env(cls):
...
+        if wipe_workspace_on_start == "True":
```
Comment (lc0rp) [inline autogpt/config/config.py]: Probably better to update config_dict here, and leave move the actual workspace path clearance to the Workspace module.
For a couple of reasons:
1. `workspace_directory` is a cli option (hidden for now, but likely to be exposed soon)
2. `build_config_from_env` shouldn't have side effects apart from initializing the config class

**Review by lc0rp — CHANGES_REQUESTED** [review]
Please write some tests

File: autogpt/config/config.py (hunk 2 above)
Comment (Pwuts) [reply autogpt/config/config.py]: Agreed, I propose adding a `wipe` parameter to `Workspace.__init__` and doing it in there.

**Pull request closed without merging**